src/networks.json

```diff
@@ -413,6 +413,20 @@
     "start": 4992871,
     "logo": "ipfs://QmSZvT9w9eUDvV1YVaq3BKKEbubtNVqu1Rin44FuN4wz11"
   },
+  "87": {
+    "key": "87",
+    "name": "Nova Network",
+    "shortName": "Nova",
+    "chainId": 87,
+    "network": "mainnet",
+    "multicall": "0x7A5a7579eb8DdEd352848cFDD0a5530C4e56FF7f",
```

Comment:
Is there a way to verify this contract?

Reply:
You can have a look here https://github.com/nova-network-inc/fusion-public-contracts/blob/main/NovaMulticall.sol the explorer isn't indexed, so doesn't support verification just yet. Working through that.